iceoryx_posh/include/iceoryx_posh/popo/notification_info.hpp

```diff
@@ -18,7 +18,7 @@
 #define IOX_POSH_POPO_NOTIFICATION_INFO_HPP
 
 #include "iceoryx_hoofs/cxx/function_ref.hpp"
-#include "iceoryx_hoofs/error_handling/error_handling.hpp"
+#include "iceoryx_posh/error_handling/error_handling.hpp"
```

Comment:
I guess here it is used in the corresponding `*.inl` file

Reply:
Moved to the `.inl`

Reply:
Hehe :grin: `.inl` for me equals a `.cpp` just for templates, to separate the implementation. Includes should always be added where they are needed. Hence, in this case including it in `.inl` is correct from my point of view. I fear we haven't documented anything about that yet.

Reply:
Don't know but you have to argue with one guy about this topic ;)